frontend/src/concepts/pipelines/content/PipelineServerActions.tsx

```diff
@@ -28,50 +30,66 @@ const PipelineServerActions: React.FC<PipelineServerActionsProps> = ({ variant,
 
   const DropdownComponent = (
     <Dropdown
-      data-testid="pipeline-server-action"
-      onSelect={() => setOpen(false)}
```

Comment:
This dropdown does not close when you select a menu item. As shown in the screenshot, the dropdown is still open after selecting the first item which opens the modal:
![image](https://github.com/opendatahub-io/odh-dashboard/assets/14068621/9f466358-e408-4b1b-aeae-5a71a14d275c)